lib/utils/check-version.js

```diff
@@ -0,0 +1,41 @@
+const request = require("request-promise");
+const Logger = require.main.require("lib/logger/logger.js");
+const Errors = require.main.require("lib/response/error.js");
+
+function CheckVersion(pkg) {
```

Comment:
If I understand correctly, the `pkg` argument should be the json-parsed contents of `package.json`?